src/common/index.ts

```diff
@@ -0,0 +1,5 @@
+import * as log from './log'
```

Comment:
Is `src/common/index.ts` file actually required anywhere? Maybe the logger could move to services.